buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -2,18 +2,18 @@
 pipeline {
     agent {label "${params.MACHINE_LABEL}"}
     parameters {
+    	string (defaultValue: "AdoptOpenJDK", description: 'personal repo', name: 'personalRepo')
+    	string (defaultValue: "master", description: 'personal branch', name: 'personalBranch')
 		choice (choices: 'x64_linux\nx64_mac\ns390x_linux\nppc64le_linux\naarch64_linux', description: 'PLATFORM?', name: 'PLATFORM')
 		choice (choices: 'SE90\nSE80', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
 		choice (choices: 'openjdk9-openj9\nopenjdk8-openj9\nopenjdk9\nopenjdk8', description: 'What is JVM Version?', name: 'JVM_VERSION')
-		string (defaultValue: 'runtest', description: 'Test Target, e.g.openjdk sanity extended test_systemtest test_DirectoryName', name: 'TARGET')
+		string (defaultValue: '', description: 'Specific test project,default is all project, e.g. openjdk_regression systemtest performance jck thirdparty_containers', name: 'TESTPROJECT')
+		string (defaultValue: 'runtest', description: 'Test Target, default is all target(runtest), e.g.openjdk sanity extended test_systemtest test_DirectoryName, jdk_bean_SE80_0, etc', name: 'TARGET')
```

Comment:
Minor rewording:
Test Target to execute, default is runtest which will run all tests.  You can also select sub-targets such as: 
  \ntop level: openjdk, sanity, extended, perf, etc 
  \ndirectory level: prefix a directory with test (eg test_systemtest) 
  \nindividual test: jdk_bean_SE80_0, etc', name: 'TARGET')
  Not sure if you can put a multi-line description using \n, but it would be nice if so...